client.cpp

```diff
@@ -332,22 +320,17 @@ PHP_METHOD(VoltClient, connect)
     }
 
     if (argc == 4) {
-        obj->client = new voltdb::Client(voltdb::ConnectionPool::pool()->acquireClient(
-                                             err, hostname,
+        obj->client = new voltdb::Client(voltdb::ConnectionPool::pool()->acquire256Client(
```

Comment:
Why do you need to use acquire256Client can you not use acquireClient with arg for hashing?